service-worker/selective-caching/service-worker.js

```diff
@@ -29,20 +29,27 @@ var CURRENT_CACHES = {
   font: 'font-cache-v' + CACHE_VERSION
 };
 
+function hashValues(hash) {
+  return Object.keys(hash).map(function(key) { return hash[key]; });
```

Comment:
no need for this function.  `Object.values(CURRENT_CACHES)` gives you the object's values.